### DISCUSSION TITLE
Windows worker nodes 'NotReady' after joining the RKE2 server; cni plugin not initialized

### DISCUSSION THREAD

**Discussion by abhinovntap** [rancher/rke2#5597] [General]
Hello Team,

I have just started setting up a RKE2 cluster with windows nodes with below config details, steps followed from the rke2 official docs. However, there are new set of CNI errors coming up after Calico is installed on the Windows VMs.

```
2 Linux(Ubuntu) nodes(1 Server and 1 agent)
2 Windows 2019 nodes(2 agents)
v1.24.13+rke2r1
Calico v3.25.0
Containerd 1.6.19
GoLang 1.19.1

RKE2-Server :kgn
NAME              STATUS     ROLES                       AGE     VERSION
s01                   Ready      control-plane,etcd,master   2d11h   v1.24.13+rke2r1
s02                   Ready      worker                      2d10h   v1.24.13+rke2r1
win-nf2tcqq   NotReady   worker                      11h     v1.24.13
win-s3dfa6h   NotReady   worker                      4h24m   v1.24.13


  Ready            False   Mon, 11 Mar 2024 02:45:17 -0400   Sun, 10 Mar 2024 22:32:40 -0400   KubeletNotReady              container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized
```
Any suggestions to fix the issue are appreciated.

**Comment by manuelbuil** [comment]
The version `v1.24.13+rke2r1` is pretty old. The Windows code has been changing quite a bit in the last releases. Could you deploy with a newer version please?

**Reply by abhinovntap** [reply]
Thank you @manuelbuil . There has been a high priority RKE2 use case with same version and further observed issues with csi-proxy service on  windows nodes, hence asked. Any particular version you'd recommend and should channel be 'stable' or 'latest' ?

**Reply by manuelbuil** [reply]
Both stable and latest are good
